pkgs/development/tools/profiling/systemtap/default.nix

```diff
@@ -0,0 +1,56 @@
+{ fetchurl, linuxPackages, makeWrapper, runCommand, fetchgit
+, elfutils, pkgconfig, gettext, stow, gnumake }:
+let
+  ## fetchgit info
+  url = git://sourceware.org/git/systemtap.git;
```

Comment:
Okay sorry, I thought it was available since the homepage states it is... :disappointed: